PWGJE/EMCALJetTasks/macros/JetQA/runJetQA.sh

```diff
@@ -21,60 +21,106 @@
+#RUNLIST="282343 282342 282341 282340 282314 282313 282312 282307 282306 282305 282304 282303 282302 282247 282230 282229 282227 282224 282206 282189 282147 282146 282126 282123 282122 282119 282118 282099 282098 282078 282051 282031 282030 282025"
+#plotDir="YourPlotDir"
+#PREFIX="/alice/data/2017/LHC17p/000"
+#SUFFIX="YourTrainSuffix"
+
+#..........................................
+#17q.......................................
+#RUNLIST="282367 282366 282365"
+#plotDir="YourPlotDir"
+#PREFIX="/alice/data/2017/LHC17q/000"
+#SUFFIX="YourTrainSuffix"
 
 downloadData=false
-plotQA=false
-generatePresentation=false
+plotQA=true
+generatePresentation=true
 
 analysisFile="AnalysisResults.root"   # For pt-hard, set to "AnalysisResultsFinal.root"
-referenceFile="../AnalysisResultsReference.root"
-
-PREFIX="/alice/data/2015/LHC15o/000"
-SUFFIX="/pass1/PWGJE/Jets_EMC_PbPb/1872_20170310-0102/AnalysisResults.root"
+#referenceFile="../AnalysisResultsReference.root"
+referenceFile="AllRuns/AnalysisResults.root"
 
+#-----------------------------------------
+# Preparations
 if [ ! -d "$plotDir" ]; then
-  mkdir "$plotDir"
+  mkdir -p $plotDir/"AllRuns"
 fi
-cd $plotDir
 echo "output dir: $plotDir"
 
+#-----------------------------------------
+# Download Files Section
 if [ "$downloadData" = true ]; then
 
-  echo "Downloading data, per run..."
+  echo "-> Downloading data, per run..."
 
   for RUN in $RUNLIST
   do
-    if [ ! -d "$RUN" ]; then
-      mkdir "$RUN"
+    if [ ! -d $plotDir/$RUN ]; then
+      mkdir "$plotDir"/"$RUN"
     fi
   done
 
   for RUN in $RUNLIST
   do
-    alien_cp alien://$PREFIX$RUN$SUFFIX $RUN
+    echo "- - - - - - - - - - - - - - - - -"
+    echo "Downloading run " $RUN
+    alien_cp alien://$PREFIX$RUN$SUFFIX $plotDir/$RUN
   done
-
 fi
 
+#-----------------------------------------
+# Plot QA of Files Section
 if [ "$plotQA" = true ]; then
 
-  echo "Executing plotting macro..."
+  echo "-> Executing plotting macro..."
+
+  echo "- - - - - - - - - - - - - - - - -"
+  echo "Plot merged reference run"
+#python $ALICE_PHYSICS/PWGJE/EMCALJetTasks/macros/JetQA/plotPWGJEQA.py -f $plotDir/$referenceFile -o $plotDir/"AllRuns"/QAoutput -i ".png"
+  python ~/Software/alice/ali-master/AliPhysics/PWGJE/EMCALJetTasks/macros/JetQA/plotPWGJEQA.py -f $plotDir/$referenceFile -o $plotDir/"AllRuns"/QAoutput -i ".png"
 
   for RUN in $RUNLIST
   do
-    python $ALICE_PHYSICS/PWGJE/EMCALJetTasks/macros/JetQA/plotPWGJEQA.py -f $RUN/$analysisFile -o $RUN/QAoutput -r $referenceFile -i ".png"
+    echo "- - - - - - - - - - - - - - - - -"
+    echo "Plotting run " $RUN
+    #python $ALICE_PHYSICS/PWGJE/EMCALJetTasks/macros/JetQA/plotPWGJEQA.py -f $plotDir/$RUN/$analysisFile -o $plotDir/$RUN/QAoutput -r $plotDir/$referenceFile -i ".png"
+    python ~/Software/alice/ali-master/AliPhysics/PWGJE/EMCALJetTasks/macros/JetQA/plotPWGJEQA.py -f $plotDir/$RUN/$analysisFile -o $plotDir/$RUN/QAoutput -r $plotDir/$referenceFile -i ".png"
```

Comment:
Don't change anything to your local paths!

Reply:
Will be changed